### PR TITLE
Update base

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR fixes test data to match new version of `base`.

### CODE REVIEW

**Comment by alexander-yevsyukov** [comment]
@dmdashenkov, @armiol, PTAL.

File: server/src/test/java/io/spine/test/validation/FakeOption.java [hunk 1]
```diff
@@ -42,20 +43,20 @@
  * thrown by the constraint produced by this option. Otherwise, the constraint never discovers any
  * violations.
  */
-public final class FakeOption extends FieldValidatingOption<Void, Object> {
+public final class FakeOption extends FieldValidatingOption<Void, Empty> {
```
Comment (dmdashenkov) [inline server/src/test/java/io/spine/test/validation/FakeOption.java]: Why `Empty`? A field option must be applicable to any kind of fields, including primitives.

Comment (alexander-yevsyukov) [reply server/src/test/java/io/spine/test/validation/FakeOption.java]: These type parameters are marked as `@ImmutableTypeParameter`. `Object` is mutable.

Comment (alexander-yevsyukov) [reply server/src/test/java/io/spine/test/validation/FakeOption.java]: It took the first matching `Message` type. It could probably be `Any`.

Comment (dmdashenkov) [reply server/src/test/java/io/spine/test/validation/FakeOption.java]: In this case, we can suppress the warning caused by `ImmutableTypeParameter` as we only use `Object`s which in fact are immutable, such as `Boolean`, `String`, `ByteString`, etc.

**Pull request merged**